WORKSPACE

```diff
@@ -362,7 +362,7 @@ dpkg_list(
 
 dpkg_src(
     name = "debian10_security",
-    package_prefix = "https://snapshot.debian.org/archive/debian-security/20191230T212302Z/",
-    packages_gz_url = "https://snapshot.debian.org/archive/debian-security/20191230T212302Z/dists/buster/updates/main/binary-amd64/Packages.gz",
-    sha256 = "20299ed97c71f26e6c34d948a358e3ac9bb9f4709a90b6f38e6094b688f5270b",
+    package_prefix = "https://snapshot.debian.org/archive/debian-security/20200219T150721Z/",
```

Comment:
somehow https://snapshot.debian.org/archive/debian-security/20200219T150721Z/ redirects to https://snapshot.debian.org/archive/debian-security/20200219T081806Z/

Reply:
It doesn't have to be same. As you noted, it will redirect to the most recent timestamp if it doesn't exist, so perhaps it's ideal to set exact timestamps instead of having all same. I remember it used to the case at some point. However, I found it cumbersome, so now I'm just using the same timestamp everywhere. As long as it works, I think this is convenient. For this reason, I do not really want to put this in a variable.

Reply:
okay that makes sense, but I don't understand where the 20200219T150721Z came from then? Is it a point in time that we chose?